common/decklist.cpp

```diff
@@ -579,8 +579,6 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         cardName.replace(rx, "'");
         rx.setPattern("Æ");
         cardName.replace(rx, "AE");
```

Comment:
I'd prefer to leave that until we know it's an issue - this should only affect much older cards